main/WebServer.cpp

```diff
@@ -964,7 +964,7 @@ namespace http {
 				if (ii == HTYPE_RaspberryGPIO)
 					bDoAdd = false;
 #endif
-				if (((ii == HTYPE_1WIRE) && (!C1Wire::Have1WireSystem())) || (ii == HTYPE_PythonPlugin))
+				if (ii == HTYPE_PythonPlugin)
```

Comment:
I have one domoticz on windows, but i using owfs which installed on raspberry, and i shared owfs path.
OWFS is always available on any OS.